generators/generator-base.js

```diff
@@ -1967,14 +1967,14 @@ module.exports = class extends PrivateBase {
     }
 
     /**
-     * get a constraint name for tables in JHipster preferred style.
+     * get a foreign key constraint name for tables in JHipster preferred style.
      *
      * @param {string} entityName - name of the entity
      * @param {string} relationshipName - name of the related entity
      * @param {string} prodDatabaseType - database type
      * @param {boolean} noSnakeCase - do not convert names to snakecase
      */
-    getConstraintName(entityName, relationshipName, prodDatabaseType, noSnakeCase) {
+    getFKConstraintName(entityName, relationshipName, prodDatabaseType, noSnakeCase) {
```

Comment:
This is our public API and hence we cant have breaking changes. Please add the below for backward compatibility
```
getConstraintName(entityName, relationshipName, prodDatabaseType, noSnakeCase) {
  getFKConstraintName(entityName, relationshipName, prodDatabaseType, noSnakeCase);
}
```

Reply:
Sure I will do that, depending the answer of next review comment.